lib/request.js

```diff
@@ -57,8 +57,15 @@ var req = exports = module.exports = {
 
 req.get =
 req.header = function header(name) {
-  var lc = name.toLowerCase();
-
+  var lc = null;
```

Comment:
You don't need to declare the `lc` variable as `null` and then assign below the `if` statement; just declare & assign the variable at once below the `if` statement :)